src/NuGet.Core/NuGet.Packaging.Core/PackageDependencyInfo.cs

```diff
@@ -39,7 +40,7 @@ public PackageDependencyInfo(PackageIdentity identity, IEnumerable<PackageDepend
         public PackageDependencyInfo(string id, NuGetVersion version, IEnumerable<PackageDependency> dependencies)
             : base(id, version)
         {
-            _dependencies = dependencies == null ? new PackageDependency[0] : dependencies.ToArray();
+            _dependencies = dependencies == null ? _empty : dependencies.ToArray();
```

Comment:
:bulb: `_dependencies?.ToArray() ?? _empty`

💭 Did `?.` support extension methods from the start? Now I forget.